### PR TITLE
Muik22

### PR DESCRIPTION
**Pull request by zachandfox**

### CODE REVIEW

File: tables/rx_potency/muik22-pot.csv [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+Muik22,BNT162_14_3dose_1mo_30,B.1.617.2 Spike:+142D+417N,Supplementary Table 3,Pseudovirus (VSV),NT50,640,1,5120,5,NULL,2022-04-11
+Muik22,BNT162_14_3dose_1mo_31,B.1.617.2 Spike:+142D+417N,Supplementary Table 3,Pseudovirus (VSV),NT50,1280,1,5120,5,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_01,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,80,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_02,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,226,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_03,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,226,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_04,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,160,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_05,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,453,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_06,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,320,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_07,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,1810,1,5120,10,NULL,2022-04-11
```
Comment (KaimingTao) [inline tables/rx_potency/muik22-pot.csv:253]: @zachandfox BNT162_01_2dose_21days_07 is not paired with any test using "hCoV-19/Belgium/rega-20174/2021", that's why the CI has errors. Two ways to solve this issue:
1. exclude the BNT162_01_2dose_21days_07 against Wildtype virus.
2. ignore the pairing of all points, just treat them as unpaired group

Comment (zachandfox) [reply tables/rx_potency/muik22-pot.csv:253]: I'm not sure what the second way would look like. I would prefer not to exclude data points if possible

File: tables/isolate_mutations.csv [hunk 2]
```diff
@@ -12281,3 +12281,51 @@ SARS-CoV-2/human/DNK/7022055836/2022,N,33,del,NULL,NULL
...
+BA.1 Spike:+214ins,S,67,V,NULL,NULL
+BA.1 Spike:+214ins,S,69,del,NULL,NULL
+BA.1 Spike:+214ins,S,70,del,NULL,NULL
+BA.1 Spike:+214ins,S,95,I,NULL,NULL
+BA.1 Spike:+214ins,S,142,D,NULL,NULL
+BA.1 Spike:+214ins,S,143,del,NULL,NULL
+BA.1 Spike:+214ins,S,144,del,NULL,NULL
+BA.1 Spike:+214ins,S,145,del,NULL,NULL
+BA.1 Spike:+214ins,S,211,del,NULL,NULL
+BA.1 Spike:+214ins,S,212,ins,NULL,NULL
```
Comment (philiptzou) [inline tables/isolate_mutations.csv:12304]: Having 212ins and 214ins so close is almost definitely an alignment issue of the paper. It should be changed to 211I + 212del + 214ins no matter what the paper suggested. If this is the case, we don't need this isolate.

File: tables/isolate_mutations.csv [hunk 3]
```diff
@@ -12281,3 +12281,51 @@ SARS-CoV-2/human/DNK/7022055836/2022,N,33,del,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,203,K,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,204,R,NULL,NULL
 SARS-CoV-2/human/DNK/7022055836/2022,N,413,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,19,R,NULL,NULL
+B.1.617.2 Spike:+142D+417N,S,142,D,NULL,NULL
```
Comment (philiptzou) [inline tables/isolate_mutations.csv:12285]: @KaimingTao too.

Can we get rid of the leading plus symbol? i.e. B.1.617.2 Spike:142D+417N. This should apply for all previous isolates.

File: tables/isolate_mutations.csv (hunk 2 above)
Comment (zachandfox) [reply tables/isolate_mutations.csv:12304]: This makes sense, I'll make the change

File: tables/rx_potency/muik22-pot.csv [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+Muik22,BNT162_01_2dose_21days_24,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,640,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_25,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,226,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_26,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,453,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_27,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,113,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_28,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,160,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_29,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,1280,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_30,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,1280,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_31,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,1810,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_32,Wuhan-Hu-1,Supplementary Table 4,Virus isolate,NT50,905,1,5120,10,NULL,2022-04-11
+Muik22,BNT162_01_2dose_21days_01,hCoV-19/Belgium/rega-20174/2021,Supplementary Table 4,Virus isolate,NT50,5,1,5120,10,NULL,2022-04-11
```
Comment (philiptzou) [inline tables/rx_potency/muik22-pot.csv:279]: So there're two types of `susc_results`. One is "linked" results, which the aggregated fold change is the median fold changes of each individual control/exp pair. The other one is "unlinked" results, which the aggregated fold change is from the geometric mean of control potency and exp potency.

What @KaimingTao suggested "the second way" is to change the `rx_name` for `hCoV-19/Belgium/rega-20174/2021` to something else that `Wuhan-Hu-1` used. For example, you cannot have `BNT162_01_2dose_21days_01` exists for both `Wuhan-Hu-1` and `hCoV-19/Belgium/rega-20174/2021` or the deriving script will try to connect them.

My suggestion is to add a suffix to the `rx_name` from line 279 to 303. For example `BNT162_01_2dose_21days_01 rega-20174`. This way will disconnect 247-278 and 279-303 -- for the deriving script to consider them as `unlinked_susc_results`.

Comment (zachandfox) [reply tables/rx_potency/muik22-pot.csv:272]: Ok, this makes sense

Comment (zachandfox) [reply tables/rx_potency/muik22-pot.csv:272]: I applied option 1 and pushed the changes before seeing this comment, should I add them back in and un-link them?

File: tables/isolates.csv [hunk 5]
```diff
@@ -1221,3 +1221,4 @@ BA.3 Spike:+417N,Omicron/BA.3,FALSE,7605591,NULL,NULL,TRUE
 SARS-CoV-2/human/DNK/7021443445/2021,B,FALSE,NULL,ON055855.1,NULL,TRUE
 SARS-CoV-2/human/DNK/7021640106/2021,Omicron/BA.1,FALSE,NULL,ON055874.1,NULL,TRUE
 SARS-CoV-2/human/DNK/7022055836/2022,Omicron/BA.2,FALSE,NULL,ON055857.1,NULL,TRUE
+B.1.617.2 Spike:142D+417N,Delta,FALSE,NULL,NULL,NULL,TRUE
```
Comment (philiptzou) [inline tables/isolates.csv:1224]: Let's use Delta/K417N.

**Pull request merged**